tests/importer/importer.cpp

```diff
@@ -76,10 +75,9 @@ TEST(Importer, warningCircularImportReferencesUnits)
 {
     const std::string errorMessage =
         "Cyclic dependencies were found when attempting to resolve units in the model 'circularImport1'. The dependency loop is:\n"
-        " - units 'i_am_cyclic' is imported from 'u2' in 'circularUnits_2.cellml';\n"
-        " - units 'u2' is imported from 'u3' in 'circularUnits_3.cellml';\n"
-        " - units 'u3' is imported from 'i_am_cyclic' in 'circularUnits_1.cellml'; and\n"
-        " - units 'i_am_cyclic' is imported from 'u2' in 'circularUnits_2.cellml'.";
+        " - the units 'i_am_cyclic' is referencing units 'u2' in the model 'circularImport2' imported from 'circularUnits_2.cellml';\n"
+        " - the units 'u2' is referencing units 'u3' in the model 'circularImport3' imported from 'circularUnits_3.cellml'; and\n"
+        " - the units 'u3' is referencing units 'i_am_cyclic' in the model 'circularImport1' imported from 'circularUnits_1.cellml'.";
```

Comment:
again, without that final message this is not describing the circular imports...